samples/contrib/versioned-pipeline-ci-samples/kaggle-ci-sample/pipeline.py

```diff
@@ -0,0 +1,41 @@
+import kfp.dsl as dsl
+import kfp.components as components
+from kfp.gcp import use_gcp_secret
+
+@dsl.pipeline(
+    name = "kaggle pipeline",
+    description = "kaggle pipeline that go from download data, train model to display result"
+)
+def kaggle_houseprice(
+    bucket_name: str,
+    commit_sha: str
+):
+    import os
```

Comment:
Question: is os used anywhere here?

Reply:
No. This is probably remnant of previous version.  Will modify.